extensions/amp-story/1.0/amp-story.css

```diff
@@ -472,6 +472,7 @@ amp-story-grid-layer .i-amphtml-embedded-component::after {
   width: var(--i-amphtml-story-layer-width, 100%);
   height: var(--i-amphtml-story-layer-height, 100%);
   font-size: calc(var(--i-amphtml-story-layer-height, 100vh) / 10);
+  margin-left: calc((var(--story-page-vw, 1%) * 100 - var(--i-amphtml-story-layer-width, 100%)) * 0.5);
```

Comment:
Why is the `margin: auto` above not centering it already? cc @processprocess

Reply:
`margin: auto` only centers horizontally if the child is smaller than the parent, because otherwise it just positions it to the left. Calculating the left margin allows it to set negative margins in that case, so it accounts for that case.

Reply:
Please add a comment above this line to explain what this math does.
We can revisit it if we find a more readable way to accomplish this layout.